src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -12,6 +13,8 @@ public class IntakeConstants {
     public static final LoggedTunableNumber ANGLE_KP = new LoggedTunableNumber("Intake/kP");
     public static final LoggedTunableNumber ANGLE_KI = new LoggedTunableNumber("Intake/kI");
     public static final LoggedTunableNumber ANGLE_KD = new LoggedTunableNumber("Intake/kD");
+    public static final double ANGLE_GEAR_RATIO = 44.44;
+    public static final TalonFXConfiguration ANGLE_CONFIGURATION = new TalonFXConfiguration();
```

Comment:
Remember to configure current limits as inversion.

Reply:
done.

Reply:
86dc3ad832e59c01d8597a47f0258055bfb998ef